src/Pos/Launcher/Param.hs

```diff
@@ -30,7 +34,17 @@ data BaseParams = BaseParams
     { bpLoggingParams   :: !LoggingParams  -- ^ Logger parameters
     } deriving (Show)
 
--- | Contains algorithm specific & storage parameters for Node.
+-- | Network parameters.
+data NetworkParams = NetworkParams
+    { npDiscovery :: !(Either (Set NodeId) KademliaParams)
+    -- ^ If this value is 'Left', then given peers will be used in static mode.
+    -- Otherwise kademlia with given params will be used.
+    , npTcpAddr   :: !TCP.TCPAddr
```

Comment:
Help message states that it is `IP and port of external address`. I will copy it to comment. I don't know if it's descriptive enough, but I don't have more information than help message and other comments in code (which are less descriptive).